apps/web/src/app/settings/registration/page.tsx

```diff
@@ -1,18 +1,21 @@
-import AccountSettings from "@/components/settings/AccountSettings";
-import { users } from "db/schema";
-import { eq } from "db/drizzle";
+import RegisterFormSettings from "@/components/settings/RegistrationForm/RegisterFormSettings";
 import { auth } from "@clerk/nextjs";
 import { db } from "db";
+import { eq } from "db/drizzle";
+import { users } from "db/schema";
 import { redirect } from "next/navigation";
 
 export default async function Page() {
 	const { userId } = auth();
+	if (!userId) throw new Error("User not found");
```

Comment:
question: Who is catching this error? This is a page so unless there is an error boundary up above, I believe this will create a client-side error. Either way, this should not throw an error to begin with. A simple redirect is what we want here.